idea_inspection.bat

```diff
@@ -0,0 +1,37 @@
+@echo off
+
+::----------------------------------------------------------------------
+:: IntelliJ IDEA inspections for checkstyle.
+::----------------------------------------------------------------------
```

Comment:
Please add example of usage

Reply:
set IDEA_HOME environment variable to be equal to the path to the idea.bat/idea.sh file for ex: 

windows: SET IDEA_HOME=C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2017.2.1\bin\idea.bat 
linux: export IDEA_JOME=/usr/local/bin/idea
mac: export IDEA_HOME="/Applications/IntelliJ IDEA.app/Contents/MacOS/idea"

(directories may be different)

than just run idea-inspection.bat/idea-inspection.sh from checkstyle root dir

Reply:
just place such examples to bat and shell file. By example user quickly see how to use it.

Please place all that bat/shell files to ./ci folder.